constants/mq.js

```diff
@@ -0,0 +1,13 @@
+export default {
+    mobile: '@media all and (max-width: 768px)',
+    tablet: '@media all and (max-width: 1024px)',
+    desktop: '@media all and (max-width: 1280px)',
+    widescreen: '@media all and (max-width: 1440px)',
+    /**
+     * @description Dynamic mq. Es. from 320px upwards.
+     * @param {string} direction 'max' as convention downwards.
```

Comment:
make this file .ts and add a proper union type for direction, it can only be min or max right?

```
type Direction = 'min' | 'max'
```